docs/internals/variable_cleanup.rst

```diff
@@ -30,23 +30,24 @@ cleans input data when it is loaded onto the stack.
 
 Different types have different rules for cleaning up invalid values:
```

Comment:
```suggestion
The following table describes the rules applied for valid and invalid values of different types, where ``higher bits`` refers to the remaining bits in case the type has less than 256 bits.
```